packages/ckeditor5-source-editing/src/sourceediting.js

```diff
@@ -0,0 +1,307 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module source-editing/sourceediting
+ */
+
+import { Plugin, PendingActions } from 'ckeditor5/src/core';
+import { ButtonView } from 'ckeditor5/src/ui';
+import { createElement, ElementReplacer } from 'ckeditor5/src/utils';
+
+import '../theme/sourceediting.css';
+
+// TODO: create icon
+// import sourceEditingIcon from '../theme/icons/sourceediting.svg';
```

Comment:
Make sure to remove these comments before the merge.

Reply:
Icon is added.